workbench/runtime.py

```diff
@@ -201,7 +201,7 @@ def get_usage_id_from_aside(self, aside_id):
         try:
             return self._aside_usages[aside_id][0]
         except KeyError:
-            raise NoSuchUsage(usage_id)
+            raise NoSuchUsage(aside_id)
```

Comment:
> This seems like a larger change.

It's not; this is me fixing a copy/paste fail.

If that line is ever run, it would break because `usage_id` is never defined in this context.

This method and several surrounding it (`get_usage_id_from_aside` and `get_aside_id_from_usage`, etc.) are nearly identical. Upon inspection, they reveal the form:

```python
def get_THING_id_from_STUFF(self, STUFF_ID):
    try:
        return self.aside_usages[STUFF_ID][INDEX]
    except KeyError:
        raise NoSuchUsage(STUFF_ID)
```

I'd bet dollars to donuts that `get_aside_type_from_usage` was written first, then copy/pasted and then they forgot to edit the line in the exception handler. Odds are either this exception is never encountered, or people just don't notice because it ends up just displaying a different kind of exception.

CC: @cpennington (original author)